docs/current/sdk/elixir/756758-get-started.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: /sdk/elixir/756758/get-started
+---
+
+# Get Started with the Dagger Elixir SDK
+
+## Introduction
+
+This tutorial teaches you the basic of using Dagger in Elixir. You will learn how to:
+
+- Install Elixir SDK
+- Create an Elixir CI task to test an application
+- Improve the Python CI tool to test the application against multiple Elixir and OTP versions
+
+## Requirements
+
+This tutorial assumes that:
+
+- You have a basic understanding of the Elixir programming language. If not, [read the Elixir learning](https://elixir-lang.org/learning.html).
+- You have a Elixir development environment with Elixir 1.14 or later and Erlang/OTP 25 or later. If not, install [Elixir](https://elixir-lang.org/install.html) and [Erlang/OTP](https://www.erlang.org/downloads).
+- You have Docker installed and running on the host system. If not, [install Docker](https://docs.docker.com/engine/install/).
+
+## Step 1: Create an Elixir application
+
+Create an Elixir project with `mix`:
```

Comment:
Should we use `application` or `project`?

Reply:
We should use project. 🙏

Reply:
Fixed